code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
-/obj/item/storage/box/syndicate/bundle_sith/PopulateContents()
-	new /obj/item/book/granter/spell/force_lightning(src)
-	new /obj/item/book/granter/martial/starter_sith(src)
-	new /obj/item/clothing/suit/armor/riot/chaplain/sith(src)
-	new /obj/item/clothing/head/helmet/chaplain/sith(src)
-	new /obj/item/melee/transforming/energy/sword/saber/sith(src)
-

```

Comment:
okay, now this (you accidentally deleting and readding the entirety of the sith martial art) is epic